src/webauth/index.js

```diff
@@ -7,15 +7,17 @@ import verifyToken from '../jwt';
 
 const {A0Auth0} = NativeModules;
 
-const callbackUri = domain => {
+const callbackUri = (domain, customScheme) => {
   const bundleIdentifier = A0Auth0.bundleIdentifier;
   const lowerCasedIdentifier = bundleIdentifier.toLowerCase();
-  if (bundleIdentifier !== lowerCasedIdentifier) {
+  if (!customScheme && bundleIdentifier !== lowerCasedIdentifier) {
```

Comment:
I'd not add this condition here. The reason being that when a custom scheme is in use, the case check for the bundle identifier (used not just in the scheme but also in the path) will not be done. And IIRC every callback URL is case sensitive. I think the warning is still helpful to point out possible casing issues. WDYT?

Reply:
That's intentional. The bundle identifier is already defined and the developer can't change it, but the custom scheme can be anything the developer wants it to be. So a forced case change is pointless, the developer should be able to specify a valid scheme from the get-go.